orca-core/src/main/groovy/com/netflix/spinnaker/orca/pipeline/persistence/jedis/JedisExecutionRepository.groovy

```diff
@@ -345,7 +345,8 @@ class JedisExecutionRepository implements ExecutionRepository {
       executingInstance: execution.executingInstance,
       status           : execution.status?.name(),
       authentication   : mapper.writeValueAsString(execution.authentication),
-      paused           : mapper.writeValueAsString(execution.paused)
+      paused           : mapper.writeValueAsString(execution.paused),
+      limitWaiting     : String.valueOf(execution.limitWaiting)
```

Comment:
the variable name here is not very clear. can we change it to something more descriptive like keepWaitingPipelines